test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -87,6 +87,13 @@
 	</test>
 	<test>
 		<testCaseName>cmdLineTester_criu_nonPortableRestore</testCaseName>
+		<disables>
+			<disable>
+				<comment>Metronome is not supported on z</comment>
```

Comment:
Please use `IBM Z` instead on z and also I would add `GC policy` to in the comment after `Metronome`.